python/perspective/perspective/nbextension/__init__.py

```diff
@@ -1,22 +0,0 @@
-#  ┏━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
```

Comment:
This file is still required

Reply:
I've moved the `PerspectiveWidget` upgrade to a dedicated branch #2658